bin/k8s-conjur-deploy/README.md

```diff
@@ -0,0 +1,12 @@
+# k8s-conjur-deploy
+This `./start` script deploy k8s Conjur environment by running `./start` of project `kubernetes-conjur-deploy`. 
```

Comment:
@sigalsax can you please review the english here?

Reply:
@moticless please don't reference `kubernetes-conjur-deploy` here. that project may change in the future and the code here will not be the same so refernecing there can confuse